changelogs/fragments/461-collection-dir.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - "The ``--collection-dir`` option was added to the wrong CLI argument parser.
+     This had no practical effect for end-users, but would also add it to other build-data based subcommands added in the future
+     (https://github.com/ansible-community/antsibull/pull/461)."
```

This doesn't describe what actually changed. Maybe something like?

```suggestion
  - "Add ``--collection-dir`` to the ``antsibull-build build-collection`` subcommand. Previously, the ``--collection-dir`` option was added to the wrong CLI argument parser and not exposed to users. (https://github.com/ansible-community/antsibull/pull/461)."
```

---

Oh, right, the consequence of how argparse operates is that the option actually wasn't there :facepalm: